dbfit-java/core/src/main/java/dbfit/api/RunnableStatement.java

```diff
@@ -0,0 +1,9 @@
+package dbfit.api;
+
+import java.sql.SQLException;
+
+public interface RunnableStatement extends AutoCloseable {
```

Comment:
> As all statements are runnable, is this a good name?

Good question, the name came from the `run` method... Basically it's statement where we don't have result returned (unlike `QueryStatement` where we have). I wondered also about `ExecutionStatement` as alternative name but it suffers from the same problem (all statements are executable).